app/controllers/admin/editions_controller.rb

```diff
@@ -417,13 +417,16 @@ def filter
   end
 
   def edition_filter_options
-    params_filters_with_default_state
+    filter_options = params_filters_with_default_state
       .symbolize_keys
       .merge(
         include_unpublishing: true,
         include_link_check_reports: true,
         include_last_author: true,
       )
+
+    filter_options = filter_options.merge(per_page: 15) if preview_design_system?(next_release: false)
```

Comment:
should the 15 be made a constant somewhere rather than hardcoded here?

Reply:
yea i think you're right. It'll likely be used across multiple pages so it makes sense to.